### ISSUE TITLE
Scrolling by clicking and dragging does not work

### ISSUE THREAD

**Issue by plufz** [ThibaultJanBeyer/DragSelect#35]
**Describe the bug**
Scrolling by clicking and dragging does not work in Chrome and Firefox - or rarely works, one time I got it to scroll in Firefox. Works fine in Safari though. I tried a couple values in autoScrollSpeed (1, 3, 100) but it didn't make any difference.

**To Reproduce**
1. Go to https://codepen.io/anon/pen/qJdgNg
2. Click and drag outside or til the edge of the window vertically

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome and Firefox
 - Version Firefox 62.0.2 and Chrome 69.0.3497.100

**Comment by ThibaultJanBeyer** [comment]
Hey, what do you mean by *almost*? When doesn’t it work?
Do you have a code example or so?

**Comment by plufz** [comment]
Of course! It does not work in Chrome when "area" is set. Works in Safari though.

Example: https://codepen.io/anon/pen/qJdgNg

**Comment by ThibaultJanBeyer** [comment]
Yes, setting the `area` to the body is a bad idea because the overflow is not on the body element but on the document element. Also the body `tag` behavior varies heavily between browsers.

**Comment by plufz** [comment]
Hi. Sorry to say but this bug is back in 1.9.1. :-) I have given you a star though!

So drag-scrolling works in version 1.9.0, but not 1.9.1 (in Chrome).
